### ISSUE TITLE
No kinematics solver has been instantiated for the group, even though the kinematics data has been loaded into the parameter server.

### ISSUE THREAD

**Issue by Sgt-Harry** [moveit/moveit2#3370]
### Description

When using the move_group_interface to control a move_group launched with ur_moveit_config, I encounter an error stating that the kinematics file (kinematics.yaml) cannot be found. However, I have verified that the kinematics parameters are loaded correctly on the parameter server.

Because

```
$ ros2 param get /move_group ur5e_manipulator.kinematics_solver
```
Output
```
String value is: kdl_kinematics_plugin/KDLKinematicsPlugin
```

### ROS Distro

Humble

### OS and version

Ubuntu 22.04 

### Source or binary build?

Binary

### If binary, which release version?

ros-humble-moveit/jammy,now 2.5.8-1jammy.20250210.235402 amd64 [installed]

### Which RMW are you using?

CycloneDDS

### Steps to Reproduce

1. Launch the move_group using ur_moveit_config.
2. Use the move_group_interface to send control commands.
3. An error is displayed indicating that the kinematics file cannot be found.

Here is my code.
<details>
<summary>The MoveGroupInterface</summary>
#include <moveit/move_group_interface/move_group_interface.h>
#include <moveit/planning_scene_interface/planning_scene_interface.h>

#include <moveit_msgs/msg/display_robot_state.hpp>
#include <moveit_msgs/msg/display_trajectory.hpp>

#include <moveit_msgs/msg/attached_collision_object.hpp>
#include <moveit_msgs/msg/collision_object.hpp>

#include <moveit_visual_tools/moveit_visual_tools.h>

static const rclcpp::Logger LOGGER = rclcpp::get_logger("move_group_logger");

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);

    auto move_group_interface_node = rclcpp::Node::make_shared("move_group_interface_node", 
      rclcpp::NodeOptions().automatically_declare_parameters_from_overrides(true));
  
    rclcpp::executors::SingleThreadedExecutor executor;
    executor.add_node(move_group_interface_node);
    std::thread([&executor]() { executor.spin(); }).detach();
  
    static const std::string PLANNING_GROUP = "ur5e_manipulator";

    moveit::planning_interface::MoveGroupInterface move_group(move_group_interface_node, PLANNING_GROUP);

    move_group.setMaxVelocityScalingFactor(0.1);
    move_group.setMaxAccelerationScalingFactor(0.1);
    move_group.setPlannerId("RRTstarkConfigDefault");

    moveit::planning_interface::PlanningSceneInterface planning_scene_interface;

    RCLCPP_INFO(LOGGER, "Planning frame: %s", move_group.getPlanningFrame().c_str());
    RCLCPP_INFO(LOGGER, "End effector link: %s", move_group.getEndEffectorLink().c_str());
    RCLCPP_INFO(LOGGER, "Available Planning Groups:");

    std::copy(move_group.getJointModelGroupNames().begin(), move_group.getJointModelGroupNames().end(),
              std::ostream_iterator<std::string>(std::cout, ", "));

    geometry_msgs::msg::PoseStamped current_pose = move_group.getCurrentPose("tool0");
    RCLCPP_INFO(LOGGER, "  Position: x=%.3f, y=%.3f, z=%.3f", 
      current_pose.pose.position.x, 
      current_pose.pose.position.y, 
      current_pose.pose.position.z);

    geometry_msgs::msg::Pose target_pose1;
    target_pose1.orientation.w = 1.0;
    target_pose1.position.x = -0.069;
    target_pose1.position.y = -0.031;
    target_pose1.position.z = 0.405;
    move_group.setApproximateJointValueTarget(target_pose1, "tool0");

    moveit::planning_interface::MoveGroupInterface::Plan my_plan;

    bool success = (move_group.plan(my_plan) == moveit::core::MoveItErrorCode::SUCCESS);

    RCLCPP_INFO(LOGGER, "Visualizing plan 1 (pose goal) %s", success ? "" : "FAILED");

    if (success) {
      RCLCPP_INFO(LOGGER, "Planning succeeded. Press 'Y' to execute, any other key to cancel.");
      char choice;
      std::cin >> choice;
      if (choice == 'Y' || choice == 'y') {
        move_group.execute(my_plan);
      }
    } else {
      RCLCPP_ERROR(LOGGER, "Planning failed.");
    }

    // END_TUTORIAL
    rclcpp::shutdown();
    return 0;
}
</details>

### Expected behavior

Runing without ERROR.

### Actual behavior

porscheharry@BINO3667:~/robot_in_car$ ros2 run robot_in_car_cpp plan_and_execute 
[INFO] [1740997473.075190570] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 1.02741 seconds
[INFO] [1740997473.075258261] [moveit_robot_model.robot_model]: Loading robot model 'ur5e'...
[INFO] [1740997473.075272951] [moveit_robot_model.robot_model]: No root/virtual joint specified in SRDF. Assuming fixed joint
[WARN] [1740997473.090552740] [moveit_ros.robot_model_loader]: No kinematics plugins defined. Fill and load kinematics.yaml!
[INFO] [1740997473.100962530] [move_group_interface]: Ready to take commands for planning group ur5e_manipulator.
[INFO] [1740997473.116518649] [moveit_ros.current_state_monitor]: Listening to joint states on topic 'joint_states'
[INFO] [1740997473.118397408] [move_group_logger]: Planning frame: world
[INFO] [1740997473.118415546] [move_group_logger]: End effector link: tool0
[INFO] [1740997473.118417838] [move_group_logger]: Available Planning Groups:
[INFO] [1740997473.120741581] [move_group_logger]:   Position: x=-0.042, y=0.032, z=0.300
[ERROR] [1740997473.122525197] [moveit_robot_state.robot_state]: No kinematics solver instantiated for group 'ur5e_manipulator'
[INFO] [1740997473.122681371] [move_group_interface]: MoveGroup action client/server ready
[INFO] [1740997473.123337826] [move_group_interface]: Planning request accepted
[INFO] [1740997478.128294512] [move_group_interface]: Planning request complete!
[INFO] [1740997478.128997289] [move_group_interface]: time taken to generate plan: 5.00154 seconds
[INFO] [1740997478.129025623] [move_group_logger]: Visualizing plan 1 (pose goal) 
[INFO] [1740997478.129038907] [move_group_logger]: Planning succeeded. Press 'Y' to execute, any other key to cancel.
ur5e_manipulator, ^C[INFO] [1740997792.983979718] [rclcpp]: signal_handler(signum=2)

**Comment by Sgt-Harry** [comment]
It turns out that the single interface node still needs to get parameters either from the running move group or directly from `rclcpp::Node::set_parameter`.

So the warning
` [moveit_robot_state.robot_state]: No kinematics solver instantiated for group 'ur5e_manipulator'`
 is gone.

However, a new issue occurs when trying to make a plan for it. Any insights on this?
`[INFO] [1741076750.671494264] [move_group_logger]: End effector: tool0
[ERROR] [1741076750.673965329] [move_group_interface]: Pose for end-effector 'tool0' not known.`

**Comment by Sgt-Harry** [comment]
Reinstall the newst deb pkg and 
set a proper proxy config solve this problem for me.
